packages/bpk-component-slider/src/BpkSlider.module.scss

```diff
@@ -19,83 +19,66 @@
 @import '~bpk-mixins/index.scss';
 
 $bpk-spacing-v2: true;
-$slider-control-inset-size: 22 * $bpk-one-pixel-rem;
-$slider-large-control-inset-size: bpk-spacing-sm() * 7;
+$slider-handle-size: bpk-spacing-base() + bpk-spacing-sm();
+$slider-handle-touch-size: bpk-spacing-xxl();
+$slider-bar-height: bpk-spacing-sm();
 
 .bpk-slider {
-  height: bpk-spacing-xl();
+  height: $slider-handle-size;
   -webkit-tap-highlight-color: transparent;
 
-  &--large {
-    height: bpk-spacing-xxl();
+  :focus-visible {
+    outline: none;
```

Comment:
Verified yes it means it's removed can you check with design we would be ok with this, especially whilst we are thinking about focus indicators this will remove this and will be not great for A11y